packages/sdk/src/tests/indexing.test.ts

```diff
@@ -0,0 +1,33 @@
+import { Indexing } from '../modules/indexing/indexing';
+import { getConfig, getIndexerUUID } from './helpers/helper';
+
+describe('Indexing tests', () => {
+  let indexing: Indexing = undefined;
+
+  beforeAll(async () => {
+    indexing = new Indexing(getConfig());
```

Comment:
you can use a variable `indexer_uuid: string` and initialize it here with `getIndexerUUID()` to avoid repeating it